src/test/java/com/puppycrawl/tools/checkstyle/filters/SuppressWithNearbyCommentFilterTest.java

```diff
@@ -469,21 +470,30 @@ public void testSuppressById() throws Exception {
     public void testTagsAreClearedEachRun() {
         final SuppressWithNearbyCommentFilter suppressionCommentFilter =
                 new SuppressWithNearbyCommentFilter();
-        final FileContents contents =
-                new FileContents("filename", "//SUPPRESS CHECKSTYLE ignore", "line2");
+        final List<?> tags1 = accept(suppressionCommentFilter,
+                "filename1", "//SUPPRESS CHECKSTYLE ignore this");
+        assertEquals("Invalid tags size", 1, tags1.size());
+        final List<?> tags2 = accept(suppressionCommentFilter,
+                "filename2", "No comments in this file");
+        assertEquals("Invalid tags size", 0, tags2.size());
+    }
+
+    /**
+     * Calls the filter with a minimal set of inputs and returns a list of
+     * {@link SuppressWithNearbyCommentFilter} internal type {@code Tag}.
+     * Testing implementation details is a bad practice, but we also need 100% test coverage.
+     * For this reason, it is necessary to use reflection to gain access to the inner field here.
+     *
+     * @return {@code Tag} list
+     */
+    private static List<?> accept(SuppressWithNearbyCommentFilter suppressWithNearbyCommentFilter,
+            String filename, String... lines) {
```

Comment:
name probably should be `getTagsAfterExecution(...)`

Reply:
done